test/evmc/mocked_host.hpp

```diff
@@ -212,7 +198,7 @@ class MockedHost : public Host
         else
             status = EVMC_STORAGE_MODIFIED_AGAIN;
```

Comment:
Hm, this means it will stay at `MODIFIED_AGAIN` even if we are resetting to zero twice, e.g.
```
original = non-zero
current = zero
value = zero
```
would be counted as `MODIFIED_AGAIN`. Is that correct?

Reply:
But given in #13532 this matches the mockedhost which I believe works with state tests, I am inclined to trust this one.

Reply:
If `current == value` then this is `EVMC_STORAGE_UNCHANGED` case but the gas cost it the same.

As described in the commit message, the only case when we don't have `MODIFIED_AGAIN` is when we loop back the `carrent` to the value of `original` (e.g. 1→2→3→1).

Reply:
> If current == value then this is EVMC_STORAGE_UNCHANGED case but the gas cost it the same.

Ah yeah, you are correct.